components/homepage/service/Services.tsx

```diff
@@ -7,21 +7,23 @@ import { Service } from '@/types';
 
 const Services = async () => {
   const services = await getServices();
-
   if (!services) {
     notFound();
   }
```

Comment:
```suggestion
  if (!services) notFound();
```
We shouldn't need these brackets.